src/Tools/dotnet-monitor/Startup.cs

```diff
@@ -29,18 +30,8 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            // AddControllers is sufficient because the tool does not use Razor nor Views.
-            services.AddControllers()
-            .AddJsonOptions(options =>
-            {
-                // Allow serialization of enum values into strings rather than numbers.
-                options.JsonSerializerOptions.Converters.Add(new JsonStringEnumConverter());
-            })
-            .AddApplicationPart(typeof(DiagController).Assembly);
-
-            services.AddControllers(options =>
-            {
-                options.Filters.Add(typeof(EgressValidationUnhandledExceptionFilter));
+            services.ConfigureHttpJsonOptions(options => {
+                options.SerializerOptions.Converters.Add(new JsonStringEnumConverter());
             });
 
             services.Configure<ApiBehaviorOptions>(options =>
```

Comment:
`ApiBehaviorOptions` seems to be an MVC concept. Could you check to see if this is required?